services/gateway/apis/http-version3/methods/modules/nft/nftSupported.js

```diff
@@ -0,0 +1,48 @@
+/*
+ * LiskHQ/lisk-service
+ * Copyright © 2023 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+
+const nftSupportedSource = require('../../../../../sources/version3/nftSupported');
+const envelope = require('../../../../../sources/version3/mappings/stdEnvelope');
+const { response, getSwaggerDescription } = require('../../../../../shared/utils');
+
+module.exports = {
+	version: '2.0',
+	swaggerApiPath: '/nft/supported',
+	rpcMethod: 'get.nft.supported',
+	tags: ['NFT'],
+	get schema() {
+		const nftSupportedSchema = {};
+		nftSupportedSchema[this.swaggerApiPath] = { get: {} };
+		nftSupportedSchema[this.swaggerApiPath].get.tags = this.tags;
+		nftSupportedSchema[this.swaggerApiPath].get.summary = 'Requests NFT Module supported collection IDs information.';
```

Comment:
The description might need an update depending on how the implementation changes.
```suggestion
		nftSupportedSchema[this.swaggerApiPath].get.summary = 'Requests NFT module supported collection IDs information.';
```

Reply:
Applied.